scripts/latexindent

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-docker run -i --rm -w "$(pwd)" -v "$(pwd):$(pwd)" $LATEXWORKSHOP_DOCKER_LATEX latexindent "$@"
+docker run -i --rm -w "$(pwd)" -v "$(pwd):$(pwd):Z" $LATEXWORKSHOP_DOCKER_LATEX latexindent "$@"
```

Comment:
what's the point of `:Z` when mapping the Docker volume?

The `-i` passed is just for executing the `latexindent "$@"` ? Questioning here if we need `-t` also (terminal)